dev/config.yaml

```diff
@@ -10,4 +12,15 @@ schemas:
   variables_path: "schemas/variables"
   interval: "5m"
 
-readonly: false
+dashboard_lists:
+  importants:
```

Comment:
by looking at the code in the frontend, it looks like this keyword is actually required to use it properly.

The config shouldn't let the user decide which name it can use if we are looking for a special keyword. The config should reflect it.
Finally by looking at the UI currently it's not possible to display other sections than the "important" one.

Did you want to provide a way to configure multiple list of important dashboard? In that case the key would be the title of the section and then you are looping other the map and create multiple card with the different dashboards.
Like we have in Grafana in Amadeus I guess

Reply:
Probably because in the current UX design we can just display a single list. Maybe we should just have list of dashboards instead of a map.
And then later we could think about moving to a map + having a proper design for that

Reply:
Removed the multi list and moved important dashboards in a "basic" list 😄